<a href="https://colab.research.google.com/github/kavitha537/Basic-ML/blob/main/My%20first%20colab%20note%20book.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

SUM OF FIRST N NATURAL NUMBERS

In [4]:
n=int(input()) 
sum=0
for i in range(1,n+1):
 sum=sum+i
 i=i+1
print(sum)

6
21


 
EXPLAINATION:
 
 
1.The above program is used to find the sum of first 'n' natural numbers. 
 
2.In this program for loop is used. 
 
3.Here n is the input. 
 
4.int represnts that input is an integer. 
 
5.Here for loop is runned between the range of i from 1 to (n+1). 
 
6.Here every value of i is multiplied with sum.And is sorted in sum. 
 
7.Here print is used to print the input
 
 8.HereI have given the input as 6.Therefore the sum of first 6 natural numbers is 21.The output is 21.

. EVEN OR ODD

In [7]:
n=int(input())
if( n%2 == 0 ):
    print("even") 
else:
    print("odd")

6
even


EXPLANATION:
 
1.The above program is to find whether the number is even Or odd. 
 
2.Let us consider the input as n. Here int represnts the input as integer. 
 
3.The condition for an integer to be an even number is " It must be divisble by 2".And the condition for odd number is " It is not divisble by 2".
 
4.So i have taken the condition that is if the number is divisible by 2 then the output will be even and if the number is not divisible by 2 then it is odd.
 
5.In this program if-else condition is used.
 
6.print is used to print the output. 
 
7.Here I have given the input as 6.since 6 is a even number. So the output is even.

FACTORIAL OF A GIVEN NUMBER

In [8]:
n=int(input())
factorial=1
for i in range(1,n+1):
 factorial=factorial*i
print(factorial)

6
720


EXPLANATION:
 
1.The above program is to find the factorial of given number. 
 
2.In this program for loop is used. 
 
3.Here n is taken as input. Int represesnts input as a integer. 
 
 
4.In this the factorial value is 1.1 is sorted in.
 
5.The range of i is taken from 1 to (n+1).
 
6.Each value of i is multiplied with the factorial which will be sorted in factorial. 
 
7.print is used to print the output after the loop is ended. 
 
8.Here I have given the input is 6.The factorial of 6 is 720.So the output is 720.